quiz.css

```diff
@@ -0,0 +1,111 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, section, summary,
+time, mark, audio, video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+  display: block;
+}
+body {
+  line-height: 1;
+}
+ol, ul {
+  list-style: none;
+}
+blockquote, q {
+  quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+
+/*
+End CSS Reset, Start Personal Formatting
+*/
+
+
+.hatCard {
+  display: inline-block;
+  border: solid 1px lightblue;
+  width: 30%;
+  min-height: 500px;
+  height: auto;
+  vertical-align: top;
+  margin: 5px;
+  padding: none;
+}
+
+.cardTitle {
+  padding: 10px;
+  border: dotted 1px lightgray;
+  text-align: center;
+  font-size: 1.2em;
+  font-weight: bold;
+}
+
+.description {
+  text-align: justify;
+  padding: 5px;
+}
+
+.bold {
+  font-size: .8em;
+  font-weight: bold;
+  margin-bottom: 4px;
+  margin-left: 10px;
+
+}
+
+.av {
+  background-color: darkgrey;
+  color: yellow;
+  margin: 15px 0px;
+  padding: 3px;
+}
+
+img {
+  max-width: 100%;
+  max-height: 150px;
+  display: block;
+  margin: 0 auto;
+}
+
+section header {
+  margin-left: 3px;
+  padding: 4px;
+}
+
+section footer {
+  font-size: .7em;
+}
```

Comment:
`section header` and `section footer` are overly specific. Use classes or simply use `header` and `footer`